src/components/MailchimpForm.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+
+import { css } from "@emotion/react";
+
+import { mediaQs } from "../mediaQueries";
+import { fonts, theme, copy  } from "../params";
+
+import addToMailchimp from 'gatsby-plugin-mailchimp';
+
+const MailchimpForm: React.FunctionComponent<{}> = () => {
+
+  const [email, setEmail] = React.useState('');
+
+  const [subscribed, setSub] = React.useState('');
```

Comment:
it would be more appropriate to use a boolean here `useState(false)`

booleans variable names are prefixed with a present tense verb

in this case that would mean `[hasSubmitted, setHasSubmitted]`